package.json

```diff
@@ -21,14 +21,14 @@
     "package": "node script/package",
     "clean:tslint": "rimraf tslint-rules/*.js",
     "compile:tslint": "tsc -p tslint-rules",
-    "lint": "npm run compile:tslint && tslint \"./app/{src,test}/**/*.{ts,tsx}\"",
-    "check-prettiness": "node script/is-it-pretty",
+    "lint": "npm run compile:tslint && npm run tslint && npm run eslint-check && npm run eslint",
+    "tslint": "tslint ./tslint-rules/*.ts ./app/{src,typings,test}/**/*.{ts,tsx}",
+    "eslint": "ts-node script/eslint.ts",
+    "eslint-check": "eslint --print-config .eslintrc.* | eslint-config-prettier-check",
     "publish": "node script/publish",
     "clean-slate": "rimraf out node_modules app/node_modules && npm install",
     "rebuild-hard:dev": "npm run clean-slate && npm run build:dev",
-    "rebuild-hard:prod": "npm run clean-slate && npm run build:prod",
-    "prettier:base": "prettier --single-quote --trailing-comma es5 --no-semi --write",
```

Comment:
> I don’t understand why it should be a separate command; could you explain that a little more?

As this currently stands, when someone gets stuck with a PR with linting issues they either need to go read some documentation or find the knowledge buried in this PR about how to fix their lint errors.

Having `npm run eslint-fix` (or equivalent) as an alias for `npm run eslint -- --fix` is quicker to type and easier to refer people to when they need to clean up their PRs is easier and friendlier (they might even find it themselves).

Is there some technical reason why this alias wouldn't work?